ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/domain-list.j2

```diff
@@ -1,6 +1,6 @@
 {# eos - domain-list #}
-{% if domain_list is defined and domain_list is not none %}
-{%    for domain in domain_list | sort %}
+{% if domain_list is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed